.github/ISSUE_TEMPLATE/submission.yaml

```diff
@@ -1,71 +1,69 @@
 ---
-name: New SDG submission
+name: New SDG proposal
 description: >-
-  Provide required information for a Small Development Grant submission.
+  Provide the required information for a new Small Development Grant proposal.
 labels:
   - Awaiting approval
 body:
   - id: instructions
     type: markdown
     attributes:
-      value: |+
```

Comment:
No obvious need to specify not normalizing trailing newlines (of which there is only the one anyway)